bundles/InstallBundle/Resources/install.sql

```diff
@@ -464,7 +464,7 @@ CREATE TABLE `search_backend_data` (
   `id` int(11) NOT NULL,
   `key` varchar(255) CHARACTER SET utf8 COLLATE utf8_bin default '',
   `index` int(11) unsigned DEFAULT '0',
-  `fullpath` varchar(765) CHARACTER SET utf8 COLLATE utf8_general_ci DEFAULT NULL, /* path in utf8 (3-byte) using the full key length of 3072 bytes */
+  `fullpath` varchar(765) DEFAULT NULL,
```

Comment:
The migration is failing because the encoding is still utf8 (while default is utf8mb4), so it needs to be converted first for existing instances/projects.

Reply:
`3-byte` is there because of the max key length of 3072 bytes, if we would go for 4-byte, we'd have to reduce  `varchar(765)` accordingly, which we can't (see `assets`, `documents` and `objects` table). 
Alsp we don't allow 4-bytes characters as keys, so that's fine and we must keep it that way.